bedrock/firefox/templates/firefox/campaign/unfck/index.html

```diff
@@ -72,27 +69,34 @@ <h3 class="c-item-title">{{ title }}</h3>
     </ul>
   {% endcall %}
 
-  <h2 class="c-section-title">The Unfcking Checklist</h2>
+  <h2 class="c-section-title">The Checklist</h2>
 
   <div class="mzp-l-content mzp-t-content-lg">
+    {% set tweet_text='2020 might be fcked, but the internet doesn’t have to be. Here are 5 easy ways we can all help turn things around.' %}
+    {% set tw_url = settings.CANONICAL_URL + '/firefox/unfck/' %}
```

Comment:
Should the tweet URL be firefox.com/unfck here?

Reply:
I think it should probably be a mz.la shortened URL that has some UTMs built into it but I haven't requested it yet.